packages/mapviewer/src/modules/map/components/openlayers/OpenLayersWMSLayer.vue

```diff
@@ -108,9 +108,9 @@ watch(projection, () => {
     layer.setSource(createSourceForProjection())
     setExtent()
 })
-watch(wmsUrlParams, () => {
-    layer.getSource().updateParams(wmsUrlParams.value)
-})
+
+watch(wmsUrlParams, layer.getSource().updateParams(wmsUrlParams.value))
+watch(() => wmsLayerConfig, setExtent, { deep: true })
```

Comment:
could we get smaller footprint by "computing" the WMS config extent, and watching it instead of the whole object?